tests/integration/fixtures/mini_sentry.py

```diff
@@ -153,6 +154,16 @@ def public_keys():
 
         return jsonify(public_keys=keys, relays=relays)
 
+    @app.route("/api/0/relays/outcomes/", methods=["POST"])
```

Comment:
Huh, we have `api/relay` and `api/0/relays` in usage, TIL. How did this happen? Not a comment on this PR, just something I just noticed

Reply:
That was very much intended: 
- `/api/relay` is for Relay's own direct API and only used for internal things like health checks. 
- `/api/0/relays/` is part of the public API and may also be served by Sentry.